src/platform-includes/session-replay/install/javascript.mdx

```diff
@@ -11,20 +11,10 @@ yarn add @sentry/browser
 ```
 
 ```html {tabTitle: CDN}
-<!--
-Note that the Replay bundle only contains the Replay integration and not the
-entire Sentry SDK. You have to add it in addition to the Sentry Browser SDK bundle:
--->
-
 <script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.tracing.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.tracing.replay.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.tracing.replay.min.js', 'sha384-base64') }}"
```

Comment:
Lovely! Do we also add the `new Replay` and sample rates now on the default `Sentry.init` call?

Reply:
Not yet, I think - I only changed the CDN bundles for now :)